src/EnergyPlus/EconomicTariff.hh

```diff
@@ -314,6 +323,8 @@ namespace EconomicTariff {
 		std::string reportMeter; // name of the report meter
 		int reportMeterIndx; // index of the report meter
 		int kindElectricMtr; // kind of electric meter - see enumerated list above, 0 is not electric
+		int kindWaterMtr; // kinf of water meter - 0 (default) is not water, 1 is water
+		int kindGasMtr; // kinf of gas meter - 0 (default) is not gas, 1 is gas
```

Comment:
What is the logic for just have this for electric, water and gas and not other types of sources?

Reply:
Because you don't have the same conversion factors between Water and Gas while you have the same underlying base unit of 'm3'

Reply:
I should have explained the logic clearly enough in the cc file using comments, see for eg [here](https://github.com/jmarrec/EnergyPlus/blob/HandleWaterUtilityCostTariffs/src/EnergyPlus/EconomicTariff.cc#L633)

I implemented the kindWAterMtr and kingGasMtr (kindElectricMtr already existing) because there was something to be done there. For the other resources, I just assumed default, for a few reasons:

* I didn't care too much about other much less used resources
* For a lot of them, you cannot infer the heat content which is necessary to convert to volumetric units. For Natural Gas, E+ assumed 1 therm = 1 CCF, which is an approximation but I'm ok with it. What about "FuelOil#1" and "FuelOil#2" (Is that actually grade 1 and 2 of the 6-scale classification used in north america?!)? Worse: OtherFuel1 and OtherFuel2?